app/presenters/two_factor_login_options_presenter.rb

```diff
@@ -59,7 +59,7 @@ def all_sms_and_voice_options_plus_one_of_each_remaining_option(configurations)
   end
 
   def presenters_for_options_that_are_not_phone(presenters)
-    presenters.select { |presenter| !phone_presenter_class?(presenter.class) }.uniq(&:class)
+    presenters.reject { |presenter| phone_presenter_class?(presenter.class) }.uniq(&:class)
```

Comment:
nice

Reply:
I found an even better way to get the final array of presenters with less code and without sacrificing readability. I was gonna submit it as a separate PR, but you're free to include it in here if you like it:

```ruby
def all_sms_and_voice_options_plus_one_of_each_remaining_option(configurations)
  presenters = configurations.flat_map(&:selection_presenters)
  phone_presenters, other_presenters = presenters.partition do |presenter| 
    phone_presenter_class?(presenter.class)
  end
  phone_presenters + other_presenters.uniq(&:class)
end
```
This would allow us to remove these methods:
```ruby
def presenters_for_options_that_are_not_phone(presenters)
  presenters.select { |presenter| !phone_presenter_class?(presenter.class) }.uniq(&:class)
end

def presenters_for_phone_options(presenters)
  presenters.select { |presenter| phone_presenter_class?(presenter.class) }
end
```

Reply:
The change was an autofix from Rubocop (`make lintfix`). I'm going to do a follow-on PR to capture the translation fixes you mentioned elsewhere. I can revisit this in that PR as well.